.github/workflows/check_formatting.yaml

```diff
@@ -80,9 +80,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.x"
-      - name: Install yapf
-        run: pip install yapf
+      - name: Install black
+        run: pip install black
         shell: bash
       - name: Check Python Formatting
-        run: yapf -r -q .
+        run: black --check .
         shell: bash
```

Comment:
For more complete format checking, you may also want to run isort with the `--check-only` option as another check here. I think it would look something like the following, matching the arguments with the pre-commit config (or just add another line to the `run:` of the existing check step):
```
- name: Check Python Formatting - isort
  run: isort --check-only --profile black --filter-files --multi-line 3
  shell: bash

Reply:
At some point the pre-commit settings need to be integrated into the GHA to replace the current formatting checks, but I don't want to rework the too much CI at the moment. That'll be a separate update.